test/fail_compilation/staticarrayoverflow.d

```diff
@@ -1,12 +1,12 @@
 /*
 REQUIRED_ARGS: -m64
 PERMUTE_ARGS:
+TEST_OUTPUT:
 ---
-fail_compilation/staticarrayoverflow.d(21): Error: static array S[1879048192] size overflowed to 7516192768000
-fail_compilation/staticarrayoverflow.d(21): Error: variable staticarrayoverflow.y size overflow
-fail_compilation/staticarrayoverflow.d(22): Error: variable staticarrayoverflow.z size of x1000ae0 exceeds max allowed size 0x100_0000
-fail_compilation/staticarrayoverflow.d(23): Error: static array S[8070450532247928832] size overflowed to 0
-fail_compilation/staticarrayoverflow.d(23): Error: variable staticarrayoverflow.a size overflow
+fail_compilation/staticarrayoverflow.d(20): Error: static array `S[1879048192]` size overflowed to 7516192768000
```

Comment:
If only the contributor to this test added TEST_OUTPUT in the first place. :-)

Need to make sure these size overflow errors aren't target specific...